### PR TITLE
Support for jest+enzyme UI tests with 1 brand new test

### PR DESCRIPTION
**Pull request by joonalillfors**

### CODE REVIEW

File: client/src/feedbacker-components/comment/comment.test.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+import { Provider } from 'react-redux'
+import Comment from './comment'
+
+let mockStore
+let testProps
+let middlewares
+
+const id = 123456
+const devComment = {
+  text: 'Wow, such a waste of talend :-DD',
```
Comment (HenrikHelenius) [inline client/src/feedbacker-components/comment/comment.test.js]: Maybe less meemeily

File: client/src/feedbacker-components/comment/comment.test.js [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+  time: '01.01.1990',
+  reactions: 'Thonking',
```
Comment (HenrikHelenius) [inline client/src/feedbacker-components/comment/comment.test.js]: Mod to correct reaction type

File: client/src/feedbacker-components/comment/comment.test.js [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+import Comment from './comment'
+
+let mockStore
+let testProps
+let middlewares
+
+const id = 123456
+const devComment = {
+  text: 'Wow, such a waste of talend :-DD',
+  time: '01.01.1990',
```
Comment (HenrikHelenius) [inline client/src/feedbacker-components/comment/comment.test.js]: Not correct timestamp format?

**Review by HenrikHelenius — CHANGES_REQUESTED** [review]

File: client/src/feedbacker-components/comment/comment.test.js (hunk 1 above)
Comment (joonalillfors) [reply client/src/feedbacker-components/comment/comment.test.js]: find

Comment (joonalillfors) [reply client/src/feedbacker-components/comment/comment.test.js]: fixxd

File: client/src/feedbacker-components/comment/comment.test.js (hunk 2 above)
Comment (joonalillfors) [reply client/src/feedbacker-components/comment/comment.test.js]: fixd

File: client/src/feedbacker-components/comment/comment.test.js (hunk 3 above)
Comment (joonalillfors) [reply client/src/feedbacker-components/comment/comment.test.js]: fixedd

File: client/src/feedbacker-components/comment/comment.test.js [hunk 4]
```diff
@@ -0,0 +1,41 @@
...
+import { Provider } from 'react-redux'
+import Comment from './comment'
+
+let mockStore
+let testProps
+let middlewares
+
+const id = 123456
+const devComment = {
+  text: 'This doesn\'t work',
```
Comment (HenrikHelenius) [inline client/src/feedbacker-components/comment/comment.test.js:13]: `` instead no need to escape

File: client/src/feedbacker-components/comment/comment.test.js [hunk 5]
```diff
@@ -0,0 +1,41 @@
...
+  time: '2019-02-02',
```
Comment (HenrikHelenius) [inline client/src/feedbacker-components/comment/comment.test.js]: hours?

**Review by HenrikHelenius — DISMISSED** [review]

File: client/src/feedbacker-components/comment/comment.test.js (hunk 4 above)
Comment (joonalillfors) [reply client/src/feedbacker-components/comment/comment.test.js:13]: eslint: Strings must use single quote :)

**Review by kalomak — APPROVED** [review]
Sick tests wow

**Review by sofiatengvall — APPROVED** [review]
Looks good to me👍👍👍👍

File: client/src/feedbacker-components/comment/comment.test.js (hunk 4 above)
Comment (HenrikHelenius) [reply client/src/feedbacker-components/comment/comment.test.js:13]: 🥰

**Review by HenrikHelenius — APPROVED** [review]
nO ChANgeS ReqUeStED 💯

**Pull request merged**